src/igvfd/schemas/changelogs/cell_line.md

```diff
@@ -1 +1,5 @@
 ## Changelog for cell_line.json
+
+### Changes since schema version 1
```

Comment:
Since there is no upgrade in this PR, should this be "Minor changes"? Do we have a guideline for this?

Reply:
fixed